docs/docs-beta/docs/integrations/libraries/looker.md

```diff
@@ -27,9 +27,35 @@ Dagster allows you to represent your Looker project as assets, alongside other y
 pip install dagster-looker
 ```
 
-### Example
+### Represent Looker assets in the asset graph
```

Comment:
Since there are multiple examples for Looker. Structuring this more like the Fivetran page